pennylane/gradients/gradient_transform.py

```diff
@@ -161,6 +161,7 @@ def jacobian_wrapper(*args, **kwargs):
             if not hybrid:
                 return qjac
 
+            kwargs.pop("shots", False)

```

Comment:
Why is this needed?

Reply:
Ah, your comment just made me think of a better way, allowing this to be removed!